main.js

```diff
@@ -170,6 +170,40 @@ function renderCartCount(cart) {
   ])
 }
 
+function renderCartItem(cartItem) {
+  return createElement('div', {class: 'row', style: 'width: 34rem; margin: 0 auto;'}, [
+    createElement('div', {class: 'col'}, [
+      createElement('img', {class: 'card-img float-left', src: cartItem.imageUrl}, [])]),
+    createElement('div', {class: 'col'}, [
+      createElement('div', {class: 'card float-right'}, [
+        createElement('h5', {class: 'card-header text-dark'}, [(cartItem.name + ' - ' + cartItem.brand)]),
+        createElement('p', {class: 'card-body'}, [cartItem.description]),
+        createElement('p', {class: 'text-success text-right mr-3'}, [('$' + cartItem.price)])
+      ])
+    ])
+  ])
+}
+
+function renderCartSummary(cart) {
+  var totalPrice = 0
+  var $cartSummary = createElement('div', {}, [
+    createElement('h2', {class: 'text-center text-primary'}, ['Cart'])
+  ])
+  var $cartList = createElement('ul', {class: 'list-group'}, [])
+  $cartSummary.appendChild($cartList)
+  for (var c = 0; c < cart.length; c++) {
```

Comment:
It would clean up this function quite a bit if you didn't calculate the total price right in the middle of rendering elements.